packages/components/package.json

```diff
@@ -68,8 +72,35 @@
     "@embroider/test-setup": "^2.1.1",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
+    "@glint/core": "^1.1.0",
+    "@glint/environment-ember-loose": "^1.1.0",
+    "@glint/template": "^1.1.0",
     "@percy/cli": "^1.24.2",
     "@percy/ember": "^4.2.0",
+    "@tsconfig/ember": "latest",
+    "@types/ember": "latest",
```

Comment:
I would suggest to use default ember types directly instead of pull this deps in

Reply:
Hey, thanks for having a look through this! I got all these `@types/*` dependencies by default on running `ember install ember-cli-typescript@latest`. Happy to try and adjust them, but I'll likely need a bit more guidance.

Reply:
yeah it gets pulled in by `ember-cli-typescript` there also possibility to use TS without `ember-cli-typescript` from version ember-cli 4.12 TS blueprints are included. I am happy to jump on the call to help with that. This week I am on EU timezone which is more aligned with you

Reply:
Oh nice, I'd be keen to pair on this! My only concern is making it work for the the handful of products still on 3.28

Reply:
I also have [PR](https://github.com/hashicorp/cloud-ui/pull/7269) that showcases how to run ember-cli v4.12 without ember-cli-typescript.

Reply:
with that setup just make sure to add `"lint:types": "tsc --noEmit",` in you `package.json` in order to run type checking as part of lint step in CI